CHANGELOG.md

```diff
@@ -4,6 +4,23 @@ _**If you are upgrading:** please see [`UPGRADING.md`](UPGRADING.md)._
 
 ## [Unreleased][unreleased]
 
+## [5.0.1] - 2019-04-07
+
+### Added
+
+- Test Electron in Travis and AppVeyor ([#607](https://github.com/Level/leveldown/issues/607)) ([**@vweevers**](https://github.com/vweevers))
+
+### Removed
+
+- Remove `slump` devDependency ([`38ff274`](https://github.com/Level/leveldown/commit/38ff274)) ([**@vweevers**](https://github.com/vweevers))
+- Remove Electron prebuilds in favor of runtime-agnostic prebuilds ([#608](https://github.com/Level/leveldown/issues/608)) ([**@vweevers**](https://github.com/vweevers))
```

Comment:
Are we good with doing this in a patch release?

Reply:
I think so. It fixes something and it's not a breaking change, right?

Reply:
It fixes Electron 4 but breaks Electron 3, that's where my doubt comes from

Reply:
Fixing Electron 4 (which is the latest) trumps Electron 3 imo.